wtxmgr/tx.go

```diff
@@ -169,72 +169,18 @@ func Create(ns walletdb.ReadWriteBucket) error {
 
 // moveMinedTx moves a transaction record from the unmined buckets to block
 // buckets.
-func (s *Store) moveMinedTx(ns walletdb.ReadWriteBucket, rec *TxRecord, recKey, recVal []byte, block *BlockMeta) error {
+func (s *Store) moveMinedTx(ns walletdb.ReadWriteBucket, rec *TxRecord,
```

Comment:
It looks like this method is no longer moving the tx from `unmined->block` buckets, but is deleting it from the unmined bucket + updating the mined balance. I think it makes sense to split it into two new methods (`deleteUnmined` and `updateMinedBalance` or similar), then call these.

Reply:
Good point! Fixed.